includes/class-newspack-newsletters-renderer.php

```diff
@@ -760,9 +955,35 @@ public static function mjml_api_credentials() {
 		return false;
 	}
 
+	/**
+	 * Expose one or more Rest routes for a public API.
+	 *
+	 * @return mixed
+	 */
+	public function register_newsletter_renderer_routes() {
+		$arguments = array(
+			/**
+			 * Return an array of properties required to render a useful ads warning.
+			 *
+			 * @uses Newspack_Newsletters_Renderer::get_ads_warning_in_editor()
+			 */
+			'callback' => array( $this, 'get_ads_warning_in_editor' ),
+			'methods'  => 'GET',
+		);
+
+		$ad_type_slug = Newspack_Newsletters_Ads::NEWSPACK_NEWSLETTERS_ADS_CPT;
+		$namespace    = "wp/v2/{$ad_type_slug}";
+		$route        = 'count';
+
+		return register_rest_route( $namespace, $route, $arguments );
```

Comment:
Registering a REST route in the renderer feels weird, I'd suggest registering the new route among the other REST routes, in `Newspack_Newsletters::rest_api_init`.

Reply:
Oh good suggestion. I didn't see that action.